extensions/namespace-lookup-management/src/main/java/io/druid/metadata/NamespaceMetadataStorageConnector.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.metadata;
+
+public interface NamespaceMetadataStorageConnector
```

Comment:
I tried to figure out how to manage the naming. Existing naming is as follows:

`SQLMetadataConnector`, `MetadataStorageTablesConfig`, `MetadataStorageConnectorConfig`

I was trying to figure out where to stick "namespace" in there somewhere.